enhancements/baremetal/baremetal-provisioning-optional.md

```diff
@@ -0,0 +1,179 @@
+---
+title: baremetal-provisioning-optional
+authors:
+  - "@stbenjam"
+reviewers:
+  - "@dhellman"
+  - "@hardys"
+  - "@kirankt"
+  - "@sadasu"
+approvers:
+  - "@hardys"
+creation-date: 2020-05-21
+last-updated: 2020-06-23
+status: provisional
+see-also:
+  - "/enhancements/baremetal/baremetal-provisioning-config.md"
+  - "/enhancements/installer/connected-assisted-installer.md"
+---
+
+# Baremetal Optional Provisioning Network
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Today the baremetal IPI platform takes a "batteries included" approach
+that has a dedicated provisioning network, HTTP caches, DHCP and TFTP
+servers, fully managed provisioning with both PXE and virtual media, and
+some level of hardware management.
```

Comment:
A reference to where the current network requirements are documented would be helpful in here somewhere

Reply:
Done